<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchDialog</class>
 <widget class="QWidget" name="SearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>283</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>2</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="searchField">
     <property name="accessibleName">
      <string>Search string</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Enter search String...</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="toggleOptions">
     <property name="accessibleName">
      <string>Options</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>.</normaloff>.</iconset>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="SearchDialogTree" name="searchResultTree">
     <property name="accessibleName">
      <string>Search results</string>
     </property>
     <property name="accessibleDescription">
      <string>Use up and down keys to navigate through the search results.</string>
     </property>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
     <column>
      <property name="text">
       <string/>
      </property>
     </column>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QWidget" name="searchOptionBox" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QCheckBox" name="userOption">
        <property name="toolTip">
         <string>Whether to search for users</string>
        </property>
        <property name="text">
         <string>&amp;Users</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="caseSensitiveOption">
        <property name="toolTip">
         <string>Whether the search should be performed case-sensitively</string>
        </property>
        <property name="text">
         <string>Case-&amp;sensitive</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="searchOptionsLabel">
        <property name="text">
         <string>Options:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QCheckBox" name="regexOption">
        <property name="toolTip">
         <string>Whether the search string should be interpreted as a regular expression</string>
        </property>
        <property name="text">
         <string>&amp;RegEx</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QCheckBox" name="channelOption">
        <property name="toolTip">
         <string>Whether to search for channels</string>
        </property>
        <property name="text">
         <string>&amp;Channels</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="searchForLabel">
        <property name="text">
         <string>Search for:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SearchDialogTree</class>
   <extends>QTreeWidget</extends>
   <header>SearchDialogTree.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>searchField</tabstop>
  <tabstop>toggleOptions</tabstop>
  <tabstop>searchResultTree</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
